x-pack/test/security_solution_cypress/serverless_config.ts

```diff
@@ -34,6 +34,9 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
           { product_line: 'endpoint', product_tier: 'complete' },
           { product_line: 'cloud', product_tier: 'complete' },
         ])}`,
+        `--xpack.securitySolution.enableExperimental=${JSON.stringify([
```

Comment:
I saw you are already passing this value to the test itself, so please, remove this configuration from here before merging :)

Reply:
same as here: https://github.com/elastic/kibana/pull/174241#discussion_r1474736863

Reply:
Please see my comment in: https://github.com/elastic/kibana/pull/174241#discussion_r1472816742